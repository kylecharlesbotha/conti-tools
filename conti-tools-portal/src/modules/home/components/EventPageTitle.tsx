import { Typography } from '@mui/material';

export const EventPageTitle = () => (
  <>
    <Typography variant="h2">Welcome to Conti Tools Portal</Typography>
  </>
);
