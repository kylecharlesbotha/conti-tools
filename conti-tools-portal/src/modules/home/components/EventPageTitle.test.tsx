import { render } from '@testing-library/react';
import { EventPageTitle } from './EventPageTitle';

test('Testing title rendering', () => {
  const { baseElement } = render(<EventPageTitle />);
  expect(baseElement).toMatchSnapshot();
});
