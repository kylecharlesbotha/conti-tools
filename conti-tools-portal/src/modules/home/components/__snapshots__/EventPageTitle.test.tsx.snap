// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing title rendering 1`] = `
<body>
  <div>
    <h2
      class="MuiTypography-root MuiTypography-h2 css-1sra7t5-MuiTypography-root"
    >
      Welcome to DBAS!
    </h2>
  </div>
</body>
`;
